src/Stratis.Bitcoin.Features.Miner.Tests/PosMintingTest.cs

```diff
@@ -351,6 +352,100 @@ public void GetNetworkWeight_UsesLast73Blocks_CalculatedNetworkWeightUsingLatest
 
             Assert.Equal(4607763.9659653762, weight);
         }
+        
+        [Fact]
+        public void CoinstakeAge_BeforeActivation_Testnet()
```

Comment:
These test names don't currently describe the behaviour being proven

Reply:
Behavior is basically described in the assertions. Each test check if at some heights coins were suitable if it's expected and not suitable if it's not expected

Reply:
It is good practice to put it in the test name, since you will see this in test runners like resharper. Otherwise we could just name test methods as test1(), test2(), test3().  The test name is your chance to provide some useful information to help locate tests when you are changing behaviours later and immediate information when a test fails.